recipes/hws/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "hws" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hws-{{ version }}.tar.gz
+  sha256: 40e5b09357e4c5038da959f7513d82ae9d3267c37435e046c6b72ac3b8518453
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - matplotlib-base
+    - h5py
+    - scikit-image
+
+test:
+  imports:
+    - HWS
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://git.ligo.org/HWS/hws
+  summary: Python toolbox for analysing Hartmann Wavefront sensor data for the LIGO and Virgo detectors
+  license: GPL-3.0
+  license_file: LICENSE.md
+
+extra:
+  recipe-maintainers:
+    - BrowneDaniel
```

This should match the GitHub account you intend to use to maintain the package in the future (e.g. updates to the recipe). So would this be correct?
```suggestion
    - danielbrown2
```